test/pseudo-tty/ref_keeps_node_running.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+require('../common');
+
+const TTY = process.binding('tty_wrap').TTY;
+const isTTY = process.binding('tty_wrap').isTTY;
+const timers = require('timers');
+
+if (isTTY(0) == false) {
+  console.log('fd 0 is not a TTY.');
```

Comment:
Yeah, the assertion is actually what I did after reading your original comment. 😄